src/test/java/dev/openfeature/sdk/integration/StepDefinitions.java

```diff
@@ -44,6 +44,7 @@ public class StepDefinitions {
     private FlagEvaluationDetails<Integer> typeErrorDetails;
 
     @BeforeAll()
+    @Given("an openfeature client is registered with cache disabled")
```

Comment:
Add the new background, it can be added to the existing `BeforeAll()`, which is nice.